docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -108,9 +108,29 @@ By default, these data streams use an {ilm-init} policy that matches their data
 For example, the data stream `metrics-system.logs-*`,
 uses the metrics {ilm-init} policy as defined in the `metrics-system.logs` index template.
 
+Want to customize your index lifecycle management? See <<data-streams-ilm-tutorial>>.
+
 [discrete]
+[[data-streams-pipelines]]
+== Ingest pipelines
+
+{agent} integrations ship with a default {ref}/ingest.html[ingest pipeline]
```

Comment:
Should we day `integrations datastreams ship with a default ...` as the relation is not 1 integration => 1 pipeline but 1 integration => n datastreams => 1 pipeline

Reply:
Good point 👍